packages/components/src/sortTabs/SortTab.js

```diff
@@ -155,20 +159,15 @@ const styles = StyleSheet.create({
     },
     marginEnd: 0,
   },
-  containerActive: {
+  containerActiveWeb: {
     backgroundColor: defaultTokens.paletteWhite,
-    web: {
-      borderBottomWidth: 2,
-      borderBottomColor: defaultTokens.paletteProductNormal,
-    },
-    ios: {
-      borderStartWidth: 4,
-      borderStartColor: defaultTokens.paletteProductNormal,
-    },
-    android: {
-      borderStartWidth: 4,
-      borderStartColor: defaultTokens.paletteProductNormal,
-    },
+    borderBottomWidth: 2,
+    borderBottomColor: defaultTokens.paletteProductNormal,
+  },
+  containerActiveMobile: {
```

Comment:
Looks like it will be necessary to also tweak container height on web.

<img width="499" alt="Screenshot 2019-05-03 at 15 31 37" src="https://user-images.githubusercontent.com/2660330/57140890-280f9d00-6db9-11e9-942c-9d731bdf4e04.png">

Reply:
<img width="596" alt="Screenshot 2019-05-06 at 09 05 40" src="https://user-images.githubusercontent.com/11021461/57210778-31c61a00-6fde-11e9-8804-0afbefa2b1f1.png">

Done!